src/js/pagination/specs/paging-collection-spec.js

```diff
@@ -345,25 +350,27 @@ define(['jquery',
 
                     it('triggers an error event when the requested page is out of range',
                         AjaxHelpers.withFakeRequests(function(requests) {
-                            var errorTriggered = false;
+                            // TODO: Re-enable expectations and remove eslint ignores
```

Comment:
Probably a later PR. There's some sort of issue with AjaxHelpers that's been around forever. I tried debugging it when it came up in this PR (because `errorTriggered` was declared but never used, with those tests commented) but couldn't figure it out easily.

@andy-armstrong, I think the AjaxHelpers issues with these tests might actually be related to the other weird $.ajax thing I showed you the other day.